_maps/RandomRuins/IceRuins/icemoon_underground_abandoned_plasma_facility.dmm

```diff
@@ -661,7 +661,7 @@
 	},
 /area/ruin/plasma_facility/commons)
 "kA" = (
-/obj/structure/closet/crate{
+/obj/structure/closet/crate/open{
 	icon_state = "crateopen"
 	},
```

Comment:
`I'd also recommend keeping the icon state to display opened so mappers can see which crates are closed and vice-versa`

I did that because you said to do this. So i remove it now?

Reply:
I guess i remove now

Reply:
If you put the `icon_state = crateopen` in the object itself instead of VV it accomplishes the same thing without a lot of manual editing.  Does that make sense?

Reply:
It's necessary for mappers!  We want the icon state change to showup while people are editing maps and ruins.  Otherwise people have to manually click and sort through VV edits which is tedious and time consuming.

![StrongDMM_18QcHc4Xfc](https://user-images.githubusercontent.com/5195984/236041320-f5086014-212d-43ce-9bd9-3d567c57ecd8.png)

Whenever there a bunch of VV edits for an object that are the same, it's generally a really good idea to make it a subtype of the object.  Here is a good example of a PR making the `welded` VV for airlocks into a helper object in #74206 .  In this case they didn't want to make it a subtype since it would have required many different subtypes of airlocks, but the helper has a visual icon to help mappers identify when an airlock is welded.

Reply:
My bad, i don't know the first thing about mapping. I'll fix that